src/metrics.rs

```diff
@@ -3,9 +3,10 @@ use std::sync::Arc;
 use atomic::{Atomic, Ordering};
 use bytemuck::NoUninit;
 
+#[non_exhaustive]
```

Comment:
Want to do `Gauge` as well?

Reply:
actually `Counter` doesn't need this since users can't construct it anyway.

Reply:
I think removing this is only thing left. Feel free to merge.